chromadb/api/__init__.py

```diff
@@ -355,6 +356,7 @@ def list_collections(
     def create_collection(
         self,
         name: str,
+        configuration: Optional[CollectionConfiguration] = None,
```

Comment:
Are we ok with this breaking the API for anyone doing positional arguments?

Reply:
Positional arguments are the devil and people need to be taught a harsh lesson.